### PR TITLE
Update FAQ - CM

### PR DESCRIPTION
**Pull request by fkaminski**
see https://contao.slack.com/archives/CK4J0KNDB/p1631515400401400

### CODE REVIEW

File: docs/manual/faq/_index.de.md [hunk 1]
```diff
@@ -166,3 +166,14 @@ die Datei `task.json`.
 Anschließend sollte der Contao Manager wieder laufen.
 {{% /expand %}}
 
+{{% expand "Kann ich die .phar Datei umbenennen?" %}}
```
Comment (fiedsch) [inline docs/manual/faq/_index.de.md]: {{% expand "Kann ich die `.phar` Datei umbenennen?" %}}

File: docs/manual/faq/_index.en.md [hunk 2]
```diff
@@ -161,3 +161,14 @@ delete the file `task.json`.
 
 After that, the Contao Manager should run again.
 {{% /expand %}}
+
+{{% expand "Can I rename the .phar file?" %}}
```
Comment (fiedsch) [inline docs/manual/faq/_index.en.md]: {{% expand "Can I rename the `.phar` file?" %}}

**Review by fiedsch — CHANGES_REQUESTED** [review]
Be consistent with the markup of filenames (=use back ticks)

File: docs/manual/faq/_index.de.md (hunk 1 above)
Comment (fritzmg) [reply docs/manual/faq/_index.de.md]: I am not sure you can use back ticks in there.

Comment (fkaminski) [reply docs/manual/faq/_index.de.md]: What about ... ».phar« ...?
see: "Kann man die URL Suffix ».html« entfernen?"

Comment (fkaminski) [reply docs/manual/faq/_index.de.md]: Done

File: docs/manual/faq/_index.en.md (hunk 2 above)
Comment (fkaminski) [reply docs/manual/faq/_index.en.md]: Done with »...«

File: docs/manual/faq/_index.de.md [hunk 3]
```diff
@@ -166,3 +166,14 @@ die Datei `task.json`.
 Anschließend sollte der Contao Manager wieder laufen.
 {{% /expand %}}
 
+{{% expand "Kann ich die ».phar« Datei umbenennen?" %}}
+Ja. Du kannst einen beliebigen Dateinamen verwenden. Allerdings ist der Contao Manager dann nicht mehr über das Contao Backend erreichbar.
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
Ja. Du kannst einen beliebigen Dateinamen verwenden. Allerdings ist der Contao Manager dann nicht mehr über das Backend erreichbar.
```

File: docs/manual/faq/_index.de.md [hunk 4]
```diff
@@ -166,3 +166,14 @@ die Datei `task.json`.
...
+In diesem Fall kannst du die [config.yml](/de/system/einstellungen/#config-yml) entsprechend anpassen. Anschließend musst du über den 
+Contao-Manager (»Systemwartung« > »Prod.-Cache erneuern«) oder über die Konsole einmalig den Anwendungs-Cache leeren.
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
Contao Manager (»Systemwartung« > »Prod.-Cache erneuern«) oder über die Konsole einmalig den Anwendungs-Cache leeren.
```

File: docs/manual/faq/_index.en.md [hunk 5]
```diff
@@ -161,3 +161,14 @@ delete the file `task.json`.
 
 After that, the Contao Manager should run again.
 {{% /expand %}}
+
+{{% expand "Can I rename the ».phar« file?" %}}
+Yes. You can use any file name you want. However, the Contao Manager is no longer accessible from the Contao Backend.
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.en.md]:
```suggestion
Yes. You can use any file name you want. However, the Contao Manager is no longer accessible from the Backend. 
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Franko

**Review by netzarbeiter — APPROVED** [review]

**Pull request merged**